### PR TITLE
New framework: Pumbaa

### PR DESCRIPTION
**Pull request by eerimoq**
Pumbaa is MicroPython on top of Simba.

The framework will generate a file called frozen.c from all .py files
found in the project src/ folder. The script file main.py is the entry
point for the user script. If main.py is missing the application will
enter the interactive Python interpreter.

All c/cpp source files (including frozen.c) will be compiled, linked
and uploaded as an ordinary c application.

http://pumbaa.readthedocs.io/en/latest/

### CODE REVIEW

File: platform.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
     "https://dl.bintray.com/platformio/dl-packages/manifest.json",
     "https://sourceforge.net/projects/platformio-storage/files/packages/manifest.json/download",
     "http://dl.platformio.org/packages/manifest.json",
-    "https://raw.githubusercontent.com/eerimoq/simba/master/make/platformio/manifest.json"
+    "https://raw.githubusercontent.com/eerimoq/simba/master/make/platformio/manifest.json",
+    "https://raw.githubusercontent.com/eerimoq/pumbaa/master/make/platformio/manifest.json"
```
Comment (ivankravets) [inline platform.json]: Do you know that you can merge 2 manifests to one?

**Comment by ivankravets** [comment]
Thanks!

File: platform.json (hunk 1 above)
Comment (eerimoq) [inline platform.json]: Nope, I did not know about that. I'll keep it in mind. Thanks!

Comment (ivankravets) [inline platform.json]: Dou you want to merge it now? I have not made a release of this platform yet. Manifest file can contain multiple packages.

Comment (eerimoq) [inline platform.json]: Go ahead and release with two manifests. I'll likely not merge the two manifests to one unless it simplifies something for you in PlatformIO.

Comment (ivankravets) [inline platform.json]: For me is not difference. It was just a hint :) 

Ok, I'll release it now

Comment (eerimoq) [inline platform.json]: Ok =)

**Pull request merged**